internal/k8s/jsonpath/value.go

```diff
@@ -40,3 +40,17 @@ func (v *Value) Set(newV reflect.Value) {
 	}
 	v.Value.Set(newV)
 }
+
+func (v *Value) Sibling(name string) (val Value, ok bool) {
+	key := reflect.ValueOf(name)
+	sib := v.parentMap.MapIndex(key)
+	if sib == (reflect.Value{}) {
```

Comment:
i think `sib.IsZero()` would be more idiomatic

Reply:
what a great error message: `panic: reflect: call of reflect.Value.IsZero on zero Value`

It looks like:
1) `Value.IsZero` returns true if it's got the value's zero value, and panics otherwise.
2) `MapIndex` returns an untyped zero value
3) `Value.IsValid` is what we want for this case